x-pack/plugins/observability_solution/apm/public/components/app/entities/logs/add_apm_callout.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { apmLight } from '@kbn/shared-svg';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
+
+export function AddAPMCallOut() {
+  const { core } = useApmPluginContext();
+
+  return (
+    <EuiPanel color="subdued" hasShadow={false}>
+      <EuiFlexGroup alignItems="center" gutterSize="s" justifyContent="flexStart">
+        <EuiFlexItem grow={0}>
+          <EuiImage
+            css={{
+              background: '#FFFFFF',
```

Comment:
Can we use the `euiTheme` for colors? Maybe `useEuiBackgroundColor` or 
from `useEuiTheme` -> I think it is e`uiTheme.colors.body`

Reply:
I will update it.